### ISSUE TITLE
Speed up

### ISSUE THREAD

**Issue by Josef-Friedrich** [Josef-Friedrich/check_zpool_scrub#3]
On older zpools with a lot of history, the check takes a long time. I found if I changed this:

	$SUDO zpool history | \
		grep "zpool scrub $1" | \
		grep -v 'zpool scrub -s' | \
		tail -n 1 | \
		awk '{print $1}'

to:

	$SUDO zpool history $1 | \
		grep "zpool scrub" | \
		grep -v 'zpool scrub -s' | \
		tail -n 1 | \
		awk '{print $1}'

it is much faster. The former does history on all zpools. The latter does a history on the requsted zpool.

**Comment by Josef-Friedrich** [comment]
```
[dan@slocum:~/src/check_zpool_scrub] $ time ./check_zpool_scrub --pool=tank_fast --warning=777600 --critical=1209600 -r
OK: The last scrub on zpool “tank_fast” was performed on 2018-08-02.05:17:45 | last_ago=295971 warning=777600 critical=1209600 progress= 20% 20% 20% 20% % 20% 20% 20% 20% speed=0 time=0

real	0m2.049s
user	0m1.734s
sys	0m0.207s
```

On older pools, history can take some time:

```
[dan@slocum:~/src/check_zpool_scrub] $ time ./check_zpool_scrub --pool=system --warning=777600 --critical=1209600 -r
OK: The last scrub on zpool “system” was performed on 2018-08-04.03:49:07 | last_ago=128662 warning=777600 critical=1209600 progress=100 speed=0 time=0

real	0m51.542s
user	0m25.648s
sys	0m3.679s
[dan@slocum:~/src/check_zpool_scrub] $ time sudo zpool history system | grep real

real	0m26.270s
user	0m13.345s
sys	0m1.804s
[dan@slocum:~/src/check_zpool_scrub] $ 
```

**Comment by Josef-Friedrich** [comment]
Should be fixed since 70ca63d33a2e4bf4553cd4707b5aa2afa9b84a95
